src/tests/eina/eina_test_error.c

```diff
@@ -220,8 +220,13 @@ EFL_START_TEST(eina_error_test_failures)
                     EINA_FALSE);
    fail_unless(ctx.did);
 
+#ifdef _MSC_VER
```

Comment:
Maybe we should use a definition tied to the platform and not the compiler. The trick part is finding one that's defined for native Windows but not on cygwin.

Reply:
I only know about `_WIN32` and `_MSC_VER`, from those `_MSC_VER` seams better. @felipealmeida do you know any other definition that we can use here that is better suited?

Reply:
Let's merge this as-is for now